apps/kitchen-sink/src/ensemble/screens/widgets.yaml

```diff
@@ -649,6 +652,17 @@ View:
                       value: 1
                     - label: Option 2
                       value: 2
+                  item-template:
+                    data: [{ name: "Close" }]
+                    name: product
+                    template:
+                      Button:
```

Comment:
Can you change this to a Text/Date input with a button? We also want to make sure users can enter values.